src/_guides/language/sound-dart.md

```diff
@@ -12,7 +12,7 @@ You'll learn how to use strong mode to enable soundness, as well as
 how to substitute types safely when overriding methods.
 
 <aside class="alert alert-info" markdown="1">
-  **Note:** The terms "sound Dart", "strong mode Dart", and "type safe Dart"
+  The terms "**sound** Dart", "**strong mode** Dart", and "**type safe** Dart"
```

Comment:
GLOBAL: I like the switch to boldface, but let's drop the quotation marks when we use bold.